include/z64scene.h

```diff
@@ -290,11 +367,11 @@ typedef struct {
     struct {
         s8 room;    // Room to switch to
         s8 bgCamIndex; // How the camera reacts during the transition. -2 for spiral staircase. -1 for generic door. 0+ will index scene CamData
-    } /* 0x0 */ sides[2]; // 0 = front, 1 = back
-    /* 0x4 */ s16   id;
-    /* 0x6 */ Vec3s pos;
-    /* 0xC */ s16   rotY;
-    /* 0xE */ u16   params;
+    } /* 0x00 */ sides[2]; // 0 = front, 1 = back
```

Comment:
Ideally this gets a typedef at some point. Also didn't we decide to remove the leading 0s when the numbers didn't require them?

Reply:
What did we decide for  `// size = 0x10`, I thought that since the total size was 2 digits, that all members should also be 2 digits? Although a quick peak though the repo and I'm seeing a lot of both cases (2 digits and 1 digits when ` // size = 0x10`?

And yeah, would be good to enum this but I'd like to wait until transition/doors/etc.. are better understood

Reply:
I think we decided to pad to the number of digits required within the struct itself rather than including the size in that.

Reply:
Also typedef-wise I meant the struct, we basically never use untyped structs.

Reply:
Oh, the internal struct? Most internal structs within structs don't seem to use a typedef, unless you wanted to move this struct externally and break this struct into two

Reply:
I think we have a few unnamed structs inside a few named structs, like a few in audio, bgCheck, camera, interface, effect, etc.
You can find them with this regex: `[^f] struct \{`